solr/core/src/java/org/apache/solr/cli/HealthcheckTool.java

```diff
@@ -173,9 +173,11 @@ protected void runCloudTool(CloudSolrClient cloudSolrClient, CommandLine cli) th
                   solrClient.request(
                       new GenericSolrRequest(
                           SolrRequest.METHOD.GET, CommonParams.SYSTEM_INFO_PATH));
-              uptime = SolrCLI.uptime((Long) systemInfo.findRecursive("jvm", "jmx", "upTimeMS"));
-              String usedMemory = (String) systemInfo.findRecursive("jvm", "memory", "used");
-              String totalMemory = (String) systemInfo.findRecursive("jvm", "memory", "total");
+              uptime =
+                  SolrCLI.uptime((Long) systemInfo._get(List.of("jvm", "jmx", "upTimeMS"), null));
+              String usedMemory = (String) systemInfo._get(List.of("jvm", "memory", "used"), null);
```

Comment:
I assume you didn't finish reviewing this PR since this shows some `_getStr` opportunities.